codespell_lib/data/dictionary.txt

```diff
@@ -11133,6 +11139,7 @@ funchtion->function
 funchtional->functional
 funchtioned->functioned
 funchtioning->functioning
+funchtionning->functioning
```

Comment:
Can you add at least the -ed and possibly the -al and maybe -s variants separately too of the double n typo.

Reply:
Added in c1d0d79